src/ipc/mount-usb-drive.ts

```diff
@@ -9,13 +9,18 @@ export interface Options {
   label?: string;
 }
 
+const MOUNT_DIRECTORY = '/media/vx/usb-drive';
```

Comment:
This is a constant which exists across all three repos...

Reply:
oof

Reply:
Now, with the script changes that @benadida suggested, this constant only exists in complete system